webapp/app/js/directives/clonejobs.js

```diff
@@ -93,6 +93,96 @@ treeherder.directive('thCloneJobs', [
     });
 
     $rootScope.$on(
+        thEvents.selectNextJob, function(ev){
+
+        var jobMap = ThResultSetModel.getJobMap($rootScope.repoName);
+        var lastJobSelected = ThResultSetModel.getSelectedJob($rootScope.repoName);
+        var el, key, job, step;
+        var step = 'nextElementSibling'
+        
+        if(lastJobSelected.el == undefined || lastJobSelected.el[0] == undefined ){
+
+            var currentElement = document.getElementsByClassName('result-set')[0];
+            el = traceJob($(currentElement),step,'job-btn');
+
+        }else{
+
+            var currentElement = lastJobSelected.el[0];    
+
+            //Parsing for parent which satisy parameter mentioned in variable step
+            while(!currentElement[step] || $(currentElement[step]).css('display') == 'none'){
+
+                if(!currentElement[step]){
+                    currentElement = traceParent($(currentElement), 1)[0];
+                }else{
+                    currentElement = currentElement[step];            
+                }
+
+            } 
+
+            //look through parent's sibling for element with class job-row
+            el = traceJob($(currentElement[step]),step,'job-btn');
+
+        }
+        
+        
+        
+        
```

Comment:
Not sure why all these new lines from 127 though 130 exist, I assume only one is needed here? There's also trailing whitespace characters here to tidy up, and on lines 135, 142, 151, 157, 161-62, 164, 166-67, 170, 173, 179, 181, and a number of others farther down the file.

Depending on what editor you use, you can set a display preference to expose trailing whitespace characters in a prominent color during coding. That will be the easiest way to tidy them up.